drivers/led/lp5569.c

```diff
@@ -83,6 +87,25 @@ static int lp5569_enable(const struct device *dev)
 		return -ENODEV;
 	}
 
+	/* flip the enable pin if specified */
+	if (config->enable_gpio.port) {
+		if (!gpio_is_ready_dt(&config->enable_gpio)) {
+			LOG_ERR("Enable gpio not ready");
+			return -ENODEV;
+		}
+
+		ret = gpio_pin_configure_dt(&config->enable_gpio,
+					    GPIO_OUTPUT_ACTIVE);
+		if (ret) {
```

Comment:
It's the same result, but it's what most drivers implement.
```suggestion
		if (ret != 0) {
```

Reply:
True. Actually looking at the rest of the driver code for lp5569 they do `ret < 0` so I'm just gonna do that as well.